packages/ai-chat/src/common/chat-model.ts

```diff
@@ -411,35 +410,35 @@ export class TextChatResponseContentImpl implements TextChatResponseContent {
 
 export class MarkdownChatResponseContentImpl implements MarkdownChatResponseContent {
     kind: 'markdownContent' = 'markdownContent';
-    protected _content: MarkdownStringImpl = new MarkdownStringImpl();
+    protected _content: string;
```

Comment:
shouldn't we keep the markdown string content? depending on its attributes we could then hand over different options to markdownit. Thereby we could allow html embeddings if someone wants to have that.